docker-compose-generator/docker-fragments/bitcoin-eclair.yml

```diff
@@ -0,0 +1,53 @@
+version: "3"
+
+services:
+  bitcoind:
+    environment:
+      BITCOIN_WALLETDIR: "/data/wallet"
```

Comment:
This will be problematic if the user switch network, as the testnet wallet will be used for mainnet and vice versa, should depends on NBITCOIN_NETWORK.

Reply:
No, it creates a sub folder with the network type

Reply:
Remove this, I will activate the wallet, but put it in a separate volumes so people who just need to resync won't dump their wallet by mistake....